test/cpp/util/test_config_cc.cc

```diff
@@ -16,21 +16,15 @@
  *
  */
 
-#include <gflags/gflags.h>
+#include "absl/flags/parse.h"
 #include "test/cpp/util/test_config.h"
 
-// In some distros, gflags is in the namespace google, and in some others,
-// in gflags. This hack is enabling us to find both.
-namespace google {}
-namespace gflags {}
-using namespace google;
-using namespace gflags;
-
 namespace grpc {
 namespace testing {
 
+// TODO(Capstan): remove `remove_flags`
 void InitTest(int* argc, char*** argv, bool remove_flags) {
-  ParseCommandLineFlags(argc, argv, remove_flags);
```

Comment:
I have not audited the 17 or so places that set `remove_flags` to `true` to see if they really needed that or not, FTR. Abseil's variant will not edit them in situ, but instead return a `std::vector<char *>` of the reduced command line ([docs](https://abseil.io/docs/cpp/guides/flags#parsing-flags-during-startup).

I could also change this function to apply the resulting edits based on `remove_flags`, and put off the audit or larger refactoring for another day.

Reply:
Chose the latter and repushed. PTAL.

Reply:
SGTM. 

One example depending on this behavior might be like

```
grpc::testing::InitTest(&argc, &argv, true);
::testing::InitGoogleTest(&argc, argv);
```